lua/lspconfig/configs/ccls.lua

```diff
@@ -1,13 +1,14 @@
 local util = require 'lspconfig.util'
 
 local function switch_source_header(bufnr)
+  local method_name = 'textDocument/switchSourceHeader'
```

Comment:
'textDocument/switchSourceHeader' or `switchsourceheader` in ccls ? @MaskRay

Reply:
It's `'textDocument/switchSourceHeader'` in both clangd and ccls. 

`switchsourceheader` is invalid.

Reply:
LGTY now?